bot/src/infrastructure/http/ZkillClient.ts

```
@@ -1,102 +1,371 @@
-import { UnifiedESIClient } from "./UnifiedESIClient";
-import { logger } from "../../lib/logger";
+import { TypeSafeHttpClient } from '../../shared/http/TypeSafeHttpClient';
+import { ZkillResponseSchema, ZkillResponse } from '../../shared/schemas/api-responses';
+import { logger } from '../../lib/logger';
+import { RateLimiter } from '../../shared/performance/rateLimiter';
+import { rateLimiterManager } from '../../shared/performance/RateLimiterManager';
+import { ValidatedConfiguration } from '../../config/validated';
+import { ExternalServiceError, ValidationError } from '../../shared/errors';
+import * as crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)